### ISSUE TITLE
Harbor Image Registry

### ISSUE THREAD

**Issue by gzm55** [badges/shields#5750]
:clipboard: **Description**

Support size and version badges for docker images hosted in private [harbor image registry].(https://github.com/goharbor/harbor)

:link: **Data**
https://github.com/goharbor/harbor

:microphone: **Motivation**

Many companies managed private harbor registries for internal usage.

**Comment by calebcartwright** [comment]
Is Harbor exclusively used for self-hosted private Docker registries?

**Comment by gzm55** [comment]
Harbor is an open source image registry, any organization can public his harbor registry. We can add the registry url on the badge api to support any harbor registry, just like jar artifact badges on a nexus service.

**Comment by calebcartwright** [comment]
Are you aware of any publicly, anonymously accessible instances? Preferably "official" or supported instances?

**Comment by gzm55** [comment]
but some images are only published to non-official public registries.

**Comment by chris48s** [comment]
@calebcartwright isn't suggesting restricting it to one registry instance, but it would be helpful to have at least one example of a public instance of this that we have some confidence is going to exist for a while for dev/integration tests

**Comment by calebcartwright** [comment]
Exactly. I also want to make sure this makes sense as an offering on Shields.io. If Harbor is only used as a private, authenticated registry for private image repositories then it sounds like it's something that could only be supported with self-hosted Shields servers, and we've historically declined to incorporate the inclusion of badges that could only be utilized with self-hosted Shields instances.
